lbrynet/extras/daemon/Daemon.py

```diff
@@ -2680,8 +2328,11 @@ def jsonrpc_channel_list(self, account_id=None, page=None, page_size=None):
         account = self.get_account_or_default(account_id)
         amount = self.get_dewies_or_error("amount", amount)
         result = await self.wallet_manager.support_claim(name, claim_id, amount, account)
-        await self.analytics_manager.send_claim_action('new_support')
-        return result
+        self.component_manager.analytics_manager.send_claim_action('new_support')
+        try:
+            return result
+        finally:
+            await self.component_manager.analytics_manager.send_claim_action('abandon')
```

Comment:
Why `abandon`?

Reply:
Will update the analytics calls today and make sure they're doing what they're supposed to